developer-workflow/sbom.rst

```diff
@@ -107,3 +107,19 @@ When removing a dependency:
    that correct package is removed from the SBOM.
 5. Commit the changes to :cpy-file:`Misc/sbom.spdx.json` and
    :cpy-file:`Tools/build/generate_sbom.py`.
+
+Updating external dependencies (cpython-source-deps)
+----------------------------------------------------
+
+Dependencies for Windows CPython builds are `stored in a separate repository <https://github.com/python/cpython-source-deps>`_
```

Comment:
Some binaries are also stored in https://github.com/python/cpython-bin-deps, though generally they should also have sources in the source-deps repo. Is this distinction important here?

Reply:
Do any of the cpython-bin-deps get shipped along with the CPython artifacts? If they're derived from the cpython-source-deps repository I think we should be okay.

Reply:
I think the only one that isn't derived from `cpython-source-deps` is `vcruntime140.dll`, which comes from our repo to make sure we always get the latest one and not whichever GHA build machine we're on.